app/models/distribution.rb

```diff
@@ -105,6 +105,7 @@ def copy_from_request(request_id)
     self.request = request
     self.organization_id = request.organization_id
     self.partner_id = request.partner_id
+    self.agency_rep = request.partner_user&.formatted_email
```

Comment:
Was this a mistake?

Reply:
I am not sure how that was even touched. It's possible that I accidentally created a space and then removed it, but I did not change the method itself.